### DISCUSSION TITLE
Version Neovim Required for nvim-cmp

### DISCUSSION THREAD

**Discussion by ph15041996** [hrsh7th/nvim-cmp#1122] [General]
Hi ,
I am trying to set up neovim development configuration in my fedora 35 machine, my neovim version installed in my system is v0.6.1, i have installed it using "sudo dnf install neovim".

i have added this below lines in my pulgin.lua file and also create cmp config file which is imported in my init.lua file.
use'neovim/nvim-lspconfig'
use'hrsh7th/cmp-nvim-lsp'
 use'hrsh7th/cmp-buffer'
use'hrsh7th/cmp-path'
use'hrsh7th/cmp-cmdline'
use'hrsh7th/nvim-cmp'

One of the  error i am getting is this
"[nvim-cmp] Your nvim does not has `nvim_create_autocmd` function. Please update to latest nvim. "

i wanted to know what version of neovim is required for the current nvim-cmp?

**Comment by hrsh7th** [comment]
nvim-cmp targets the latest stable neovim.

Therefore, the current version is `v0.7.2` or later.

**Comment by ph15041996** [comment]
Thanks, it is working on v0.7.2
